src/coreclr/vm/eepolicy.cpp

```diff
@@ -67,8 +67,8 @@ void SafeExitProcess(UINT exitCode, ShutdownCompleteAction sca = SCA_ExitProcess
             {
```

Comment:
Nit: You can delete the check for `aspnet_compiler` a few lines above. I am sure we do not need that for .NET Core :)